imnodes-wgpu-examples/src/main.rs

```diff
@@ -39,19 +39,12 @@ fn main() {
     }))
     .unwrap();
 
-    let (device, queue) = block_on(adapter.request_device(
-        &wgpu::DeviceDescriptor {
-            features: wgpu::Features::empty(),
-            limits: wgpu::Limits::default(),
-            shader_validation: false,
-        },
-        None,
-    ))
-    .unwrap();
+    let (device, queue) =
+        block_on(adapter.request_device(&wgpu::DeviceDescriptor::default(), None)).unwrap();
```

Comment:
I'm not 100% sure if these lines are equivalent, but it looks like the `Default` trait wasn't implemented for `wgpu::DeviceDescriptor` at that time. So now it is possible to just do `wgpu::DeviceDescriptor::default()`